.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Let's follow GitHub's [recommendations](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings?platform=linux#example) just to be on the safe side.

```suggestion
# Set the default behavior, in case people don't have core.autocrlf set.
* text=auto

# Explicitly declare text files you want to always be normalized and converted
# to native line endings on checkout.
*.c text
*.h text

# Declare files that will always have CRLF line endings on checkout.
*.sln text eol=crlf

# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary

```

And having a newline at the EOF is also good to be POSIX compliant.